Detectors/ITSMFT/ITS/tracking/src/TrackerTraits.cxx

```diff
@@ -923,27 +925,20 @@ track::TrackParCov TrackerTraits::buildTrackSeed(const Cluster& cluster1, const
   const float y3 = tf3.positionTrackingFrame[0];
   const float z3 = tf3.positionTrackingFrame[1];
 
-  const float crv = math_utils::computeCurvature(x1, y1, x2, y2, x3, y3);
-  const float x0 = math_utils::computeCurvatureCentreX(x1, y1, x2, y2, x3, y3);
+  const float crv = math_utils::computeCurvature(x3, y3, x2, y2, x1, y1);
+  const float x0 = math_utils::computeCurvatureCentreX(x3, y3, x2, y2, x1, y1);
   const float tgl12 = math_utils::computeTanDipAngle(x1, y1, x2, y2, z1, z2);
   const float tgl23 = math_utils::computeTanDipAngle(x2, y2, x3, y3, z2, z3);
 
-  const float fy = 1. / (cluster2.radius - cluster3.radius);
-  const float& tz = fy;
-  float cy = 1.e15f;
-  if (std::abs(getBz()) > o2::constants::math::Almost0) {
-    cy = (math_utils::computeCurvature(x1, y1, x2, y2 + resolution, x3, y3) - crv) /
-         (resolution * getBz() * o2::constants::math::B2C) *
-         20.f; // FIXME: MS contribution to the cov[14] (*20 added)
-  }
-  const float s2 = resolution;
-
   return track::TrackParCov(tf3.xTrackingFrame, tf3.alphaTrackingFrame,
                             {y3, z3, crv * (x3 - x0), 0.5f * (tgl12 + tgl23),
```

Comment:
Why `crv` is still used with B=0?
Could you comment the reason of the clusters permutations above? Most important: before you were setting tight errors in the seed cov.matrix, also to slopes, which makes me to think that at least 2 updates were accounted already on the seed level. Now they are relaxed, is this related to the permutations above?

Reply:
Got your update after my comment. I assume the plot is for B!=0 ? Do you have it for B=0?

Reply:
Hi @shahor02, here is the same plot before and after this PR as it is now for B=0.
<img width="649" alt="image" src="https://user-images.githubusercontent.com/2094113/201963361-9e504465-4adb-4063-9ef7-9a6797c83205.png">
In the old code, two-track updates were approximated, that was the Cooked Matrix initialisation by @iouribelikov.
The permutation of the clusters is done just to keep the same signature of the function, that in the future might also disappear.

Reply:
ok, thanks, looks good, just see my question above about using crv for snp estimate also in case of B=0.

Reply:
Right, I can also avoid it for B=0. Is `DeltaY / std::hypot(DeltaX, DeltaY)` enough for the sinPhi?

Reply:
it would be more precise:
```
auto tgp = std::atan2(DeltaY, DeltaX);
snp = tgp / std::sqrt(1 + tgp*tgp);
```
BTW, I guess one should add a fix for the GPU as well?